cli/internal/packager/injector.go

```diff
@@ -21,7 +21,7 @@ import (
 // We may need to make this dynamic in the future
 var payloadChunkSize = 1024 * 512
 
-func runInjectionMadness(tempPath tempPaths) {
+func runCrazyConfigmapInjector(tempPath tempPaths) {
 	message.Debugf("packager.runInjectionMadness(%v)", tempPath)
```

Comment:
nit; the function name has changed

Reply:
hahe well I thought it was confusing not stating what _type_ of injection it was since I had to add the other one....